libs/design-system/src/panda/recipes/text.recipe.ts

```diff
@@ -0,0 +1,22 @@
+import { defineRecipe } from '@pandacss/dev';
+
+export const TEXT_RECIPE = defineRecipe({
+  className: 'text',
```

Comment:
What would you suggest?

Reply:
`novu-` or something shorter like `nv-` wdyt?

Reply:
https://panda-css.com/docs/references/config#prefix

Reply:
I believe this `className` directly impacts how the recipe is generated, so it may have to be imported then as `nvText` which I would rather not have. However, if using the `prefix` from the config just namespaces the CSS itself, I'm all for it.

Reply:
that's what I understand that it should only prefix the generated css classes